app/models/bank_transaction.rb

```diff
@@ -1,2 +1,4 @@
 class BankTransaction < ApplicationRecord
+    def self.get_transactions_for_time_period(date_from, date_to)
+    end
```

Comment:
In ruby werden keine getter und setter versendet, also arbeiten wir einfach mit einem named scope ;)
```suggestion
  def self.for_time_period(start_date, end_date)
    where('transaction_time >= ? AND transaction_time <= ?', start_date, end_date)
  end
```